scalafix-tests/output/src/main/scala/test/redundantSyntax/StringInterpolator.scala

```diff
@@ -0,0 +1,25 @@
+package test.redundantSyntax
+
+class StringInterpolator {
+
+  val a = 42d
+
+  var b = ""
+  b = "foo"
+  b = "foo"
+  b = s"foo $a bar"
+
+  b = """foo"""
+  b =
+    """foo
+       |bar"""
+  b = s"""foo $a bar"""
+  b = s"""$a"""
+
+  b = "foo"
+  b = f"foo $a%2.2f"
+
+  b = raw"foo $a \nbar"
+  b = raw"foo\nbar"
+  b = raw"foo bar"
```

Comment:
I found a solution to remove this one
```suggestion
  b = "foo bar"
```

with [a new condition in the rule](url):

```diff
-if config.stringInterpolator && (p == "s" || p == "f") =>
+if config.stringInterpolator && (p == "s" || p == "f" || (p == "raw" && StringContext.processEscapes(content) == content))
```

WDYT?